### DISCUSSION TITLE
nginx fulltest 403 forbidden error

### DISCUSSION THREAD

**Discussion by KrishCrS** [open-quantum-safe/oqs-demos#251] [Q&A]
Hello,
1. I followed the **[oqs-nginx fulltest](https://github.com/open-quantum-safe/oqs-demos/tree/main/nginx/fulltest)**, and it is working fine in **Ubuntu 20.04.6 LTS**, but while I was doing the same in **Ubuntu 22.04.3 LTS**, I got the `403 Forbidden error`. Is there a version issue or something else?
    **80-error.log :**
        `....../nginx/html/index-base.html" is forbidden (13: Permission denied),......`
        `....../nginx/html/favicon.ico" failed (13: Permission denied),.......`
        I checked all the permissions on both PCs, and those were the same.
2. Can't we use `dilithium` instead of `rsa` in the case of the CA certificate? Also, I noticed that the **ca cert**, generated in qs-openvpn (generated using `dilithium`), is not getting imported by qs-chromium. Why?

**Comment by baentsch** [comment]
> ../nginx/html/index-base.html" is forbidden

I'm afraid we don't have the bandwidth to dig into file permission errors on platforms we don't support. If you find a general setup error, please report it, ideally via PR to fix it.

> ../nginx/html/index-base.html" is forbidden

Tagging @pi-314159 for input

**Reply by KrishCrS** [reply]
Okay.
What about the second one? Can qs-chromium import a CA certificate generated using `dilithium`?

**Reply by baentsch** [reply]
That I don't know, hence tagged @pi-314159.

**Reply by Raytonne** [reply]
@KrishCrS Please check #92 

Chrome Root Store's root certificates = root certificates from the component updater + root certificates from the local trust store.

You can't import dilithium root certificates to the Chromium because the local trust store (i.e. libnss) doesn't support dilithium certificates.

However, you can still add this dilithium root to the Chrome Root Store by modifying the component updater. If you know component updater well, this modification is not difficult.
